src/v2/views/FormView.js

```diff
@@ -36,10 +37,26 @@ define([
         }
       }).last();
 
-      this.listenTo(form, 'save', (model) => {
-        this.options.appState.trigger('saveForm', model);
-      });
+      this.listenTo(form, 'save', this.saveForm);
+
+      this.maybeRunPolling(remediationValue, model);
```

Comment:
can we re-name this function name ? 
`this.checkAndExecutePolling`
`this.checkAndDoPolling`
`this.doPolling`
`this.doPolling`